DataFormats/BTauReco/src/classes_def.xml

```diff
@@ -410,18 +414,24 @@
     <version ClassVersion="3" checksum="386672900"/>
     <version ClassVersion="4" checksum="113477759"/>
   </class>
-  <class name="btagbtvdeep::SecondaryVertexFeatures" ClassVersion="4">
+  <class name="btagbtvdeep::SecondaryVertexFeatures" ClassVersion="6">
+   <version ClassVersion="6" checksum="1869636758"/>
+   <version ClassVersion="5" checksum="1572623975"/>
    <version ClassVersion="4" checksum="3987593676"/>
     <version ClassVersion="3" checksum="3610780843"/>
   </class>
   <class name="btagbtvdeep::ShallowTagInfoFeatures" ClassVersion="3">
     <version ClassVersion="3" checksum="273372007"/>
   </class>
-  <class name="btagbtvdeep::NeutralCandidateFeatures" ClassVersion="4">
+  <class name="btagbtvdeep::NeutralCandidateFeatures" ClassVersion="6">
+   <version ClassVersion="6" checksum="267457446"/>
+   <version ClassVersion="5" checksum="2915967773"/>
    <version ClassVersion="4" checksum="3657612864"/>
     <version ClassVersion="3" checksum="938442516"/>
   </class>
-  <class name="btagbtvdeep::ChargedCandidateFeatures" ClassVersion="5">
+  <class name="btagbtvdeep::ChargedCandidateFeatures" ClassVersion="7">
+   <version ClassVersion="7" checksum="3294985591"/>
+   <version ClassVersion="6" checksum="2366727816"/>
```

Comment:
This intermediate "6" version is not needed -> move "7" to "6"

Reply:
Same as for https://github.com/cms-sw/cmssw/pull/40706#discussion_r1099056341

And should be resolved with https://github.com/cms-sw/cmssw/pull/40706/commits/6727ef6e2ed76f0659c1b494adfeda38a53075e1